ManageCoursesUi.Tests/Controllers/OrganisationControllerTests.cs

```diff
@@ -608,7 +606,7 @@ public void AboutPost_ModelState_WordCount()
                 .ReturnsAsync(institutionCourses);
 
 
-            var enrichmentModel = new InstitutionEnrichmentModel { AccreditingProviderEnrichments = new ObservableCollection<AccreditingProviderEnrichment> { } };
+            var enrichmentModel = new InstitutionEnrichmentModel { AccreditingProviderEnrichments = new List<AccreditingProviderEnrichment> { } };
```

Comment:
hurrah!

Reply:
what wrong? you wanted Observable`PainOfA`Collection back